### ISSUE TITLE
Dev Issue: currently unable to import layers locally ( suspect: postgis 2.4)

### ISSUE THREAD

**Issue by emilyashley** [MapStory/mapstory#1163]
@goudine and I are both unable to import layers locally today. I pulled and rebuilt the docker containers on Friday, but unfortunately am unsure when the last time I did that was so this may have been an issue for a minute?

<img width="669" alt="screen shot 2018-01-08 at 5 31 17 pm" src="https://user-images.githubusercontent.com/15912063/34697699-e244a6ce-f499-11e7-8035-0fb00fc34c46.png">

**Comment by emilyashley** [comment]
From our parking lot today
### jsconsole:
<img width="1422" alt="screen shot 2018-01-08 at 5 21 26 pm" src="https://user-images.githubusercontent.com/15912063/34697395-83841968-f498-11e7-9755-535985daa76c.png">


### network tab:
<img width="966" alt="screen shot 2018-01-08 at 5 21 43 pm" src="https://user-images.githubusercontent.com/15912063/34697409-8b22705c-f498-11e7-9970-7f1947047b0f.png">

### celery logs:

```
celery log: celery_1                | Layer import task failed, recording UploadLayer.import_status
celery_1                | [2018-01-08 16:43:53,998: INFO/Worker-2] Layer import task failed, recording UploadLayer.import_status
celery_1                | [2018-01-08 16:43:54,011: ERROR/MainProcess] Task osgeo_importer.tasks.import_object[662cc986-9636-4d66-989e-f1b36be006b7] raised unexpected: RuntimeError('ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory\n',)
celery_1                | Traceback (most recent call last):
celery_1                |   File "/usr/local/lib/python2.7/site-packages/celery/app/trace.py", line 240, in trace_task
celery_1                |     R = retval = fun(*args, **kwargs)
celery_1                |   File "/usr/local/lib/python2.7/site-packages/celery/app/trace.py", line 438, in __protected_call__
celery_1                |     return self.run(*args, **kwargs)
celery_1                |   File "/srv/mapstory/deps/django-osgeo-importer/osgeo_importer/tasks.py", line 90, in import_object
celery_1                |     gi.handle(configuration_options=configuration_options, request_cookies=request_cookies, request_user=request_user)
celery_1                |   File "/srv/mapstory/deps/django-osgeo-importer/osgeo_importer/importers.py", line 115, in handle
celery_1                |     layers = self.import_file(configuration_options=configuration_options)
celery_1                |   File "/srv/mapstory/deps/django-osgeo-importer/osgeo_importer/importers.py", line 362, in import_file
celery_1                |     target_file, _ = self.open_target_datastore(self.target_store)
celery_1                |   File "/srv/mapstory/deps/django-osgeo-importer/osgeo_importer/importers.py", line 188, in open_target_datastore
celery_1                |     return self.open_datastore(connection_string, self.target_inspectors, *args, **kwargs)
celery_1                |   File "/srv/mapstory/deps/django-osgeo-importer/osgeo_importer/importers.py", line 149, in open_datastore
celery_1                |     data = insp.open()
celery_1                |   File "/srv/mapstory/deps/django-osgeo-importer/osgeo_importer/inspectors.py", line 68, in open
celery_1                |     self.data = ogr.Open(self.connection_string, 1)
celery_1                |   File "/usr/local/lib/python2.7/site-packages/osgeo/ogr.py", line 7923, in Open
celery_1                |     return _ogr.Open(*args, **kwargs)
celery_1                | RuntimeError: ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory 
```
### postgres logs:

```
   | ERROR:  database "mapstory_geogig" already exists
postgres_1              | STATEMENT:  CREATE DATABASE mapstory_geogig;
postgres_1              | LOG:  invalid length of startup packet
postgres_1              | LOG:  using stale statistics instead of current ones because stats collector is not responding
postgres_1              | LOG:  using stale statistics instead of current ones because stats collector is not responding
postgres_1              | LOG:  invalid length of startup packet
postgres_1              | LOG:  invalid length of startup packet
postgres_1              | LOG:  invalid length of startup packet
postgres_1              | LOG:  invalid length of startup packet
postgres_1              | ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
postgres_1              | STATEMENT:  SELECT postgis_version()
postgres_1              | ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
postgres_1              | STATEMENT:  SELECT postgis_version()
postgres_1              | ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
postgres_1              | STATEMENT:  SELECT postgis_version()
postgres_1              | ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
postgres_1              | STATEMENT:  SELECT postgis_version()
postgres_1              | ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
postgres_1              | STATEMENT:  SELECT postgis_version()
postgres_1              | ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
postgres_1              | STATEMENT:  SELECT postgis_version()
postgres_1              | ERROR:  could not access file "$libdir/postgis-2.3": No such file or directory
postgres_1              | STATEMENT:  SELECT postgis_version()
postgres_1              | ERROR:  database "mapstory" already exists
postgres_1              | STATEMENT:  CREATE DATABASE mapstory;
postgres_1              | ERROR:  extension "postgis" already exists
postgres_1              | STATEMENT:  CREATE EXTENSION postgis;
postgres_1              | ERROR:  extension "postgis_topology" already exists
postgres_1              | STATEMENT:  CREATE EXTENSION postgis_topology;
postgres_1              | ERROR:  database "mapstory_data" already exists
postgres_1              | STATEMENT:  CREATE DATABASE mapstory_data;
postgres_1              | ERROR:  extension "postgis" already exists
postgres_1              | STATEMENT:  CREATE EXTENSION postgis;
postgres_1              | ERROR:  extension "postgis_topology" already exists
postgres_1              | STATEMENT:  CREATE EXTENSION postgis_topology;
postgres_1              | ERROR:  database "mapstory_geogig" already exists
postgres_1              | STATEMENT:  CREATE DATABASE mapstory_geogig;
```

**Comment by cuttlefish** [comment]
[edited title. issue is with the postgis version, not the postgres version]

**Comment by cuttlefish** [comment]
Should be resolved by commit e7ada5e0bb81a1aa52fec94ef83dde7e5f00c4f1

**Comment by emilyashley** [comment]
note to self: `dco kill pgadmin; dco rm -f pgadmin; dco build --pull pgadmin; dco up -d pgadmin`
